geoips/plugins/modules/readers/modis_hdf4.py

```diff
@@ -482,7 +488,8 @@ def call(fnames, metadata_only=False, chans=None, area_def=None, self_register=F
                         if datasettag == "QKM":
                             factor = 4
                         outdata = np.zeros(
-                            (len(data) * factor, data.shape[1] * factor), data.dtype
+                            (len(data) * factor, data.shape[1] * factor),
```

Comment:
See above comment about my formatting concerns

Reply:
This is addressed in [this](https://github.com/NRLMMD-GEOIPS/geoips/commit/4f4fba4299d8f55681689e9369428c4cbc56e385) commit